diff -up elinks-0.19.1/src/config/options.c.4~ elinks-0.19.1/src/config/options.c
--- elinks-0.19.1/src/config/options.c.4~	2026-02-07 08:23:38.000000000 +0100
+++ elinks-0.19.1/src/config/options.c	2026-02-07 17:43:09.708090112 +0100
@@ -802,9 +802,10 @@ register_autocreated_options(void)
 	get_opt_bool("terminal.linux.m11_hack", NULL) = 1;
 	get_opt_int("terminal.vt100.type", NULL) = TERM_VT100;
 	get_opt_int("terminal.vt110.type", NULL) = TERM_VT100;
-	get_opt_int("terminal.xterm.type", NULL) = TERM_VT100;
+	get_opt_int("terminal.xterm.type", NULL) = 2;
 	get_opt_bool("terminal.xterm.underline", NULL) = 1;
 	get_opt_bool("terminal.xterm.strike", NULL) = 1;
+	get_opt_int("terminal.xterm.colors", NULL) = 1;
 	get_opt_int("terminal.xterm-color.type", NULL) = TERM_VT100;
 	get_opt_int("terminal.xterm-color.colors", NULL) = COLOR_MODE_16;
 	get_opt_bool("terminal.xterm-color.underline", NULL) = 1;
